### ISSUE TITLE
Error importing 

### ISSUE THREAD

**Issue by kshrestha99** [Keyang/node-csvtojson#418]
I have done this:
npm i csvtojson -D
npm i @types/csvtojson -D

In my typescript file:
```
import * as csvtojson from 'csvtojson;

csvtojson().fromFile(csvFilePath)
```

In my console:
Error Message: Error: TSError: ⨯ Unable to compile TypeScript:
someTypeScriptFile.ts:35:11 - error TS2349: This expression is not callable.
Type '{ default: (param?: Partial, options?: TransformOptions) => Converter; }' has no call signatures.

**Comment by abettermap** [comment]
Not sure if this is your problem, but you're missing a `'` before the semicolon:

```ts
import * as csvtojson from 'csvtojson;
```

Should be:

```ts
import * as csvtojson from 'csvtojson';
```

**Comment by abettermap** [comment]
...which still doesn't work 😆 

But this does:

```ts
import csvtojson from 'csvtojson';
```

**Comment by udondan** [comment]
Yes this works, but only if you set `esModuleInterop": true` in yout tsconfig. Haven't found another way.
